cdc/model/sink.go

```diff
@@ -269,6 +270,15 @@ type RowChangedEvent struct {
 
 	// SplitTxn marks this RowChangedEvent as the first line of a new txn.
 	SplitTxn bool `json:"-" msg:"-"`
+	// This filed is only used by expression filter.
+	// It will be set to nil after used.
+	RowChangedDatums *RowChangedDatums `json:"-" msg:"-"`
+}
+
+// RowChangedDatums is used to store the changed datums of a row.
+type RowChangedDatums struct {
+	RowDatums    []types.Datum `json:"-" msg:"-"`
+	PreRowDatums []types.Datum `json:"-" msg:"-"`
```

Comment:
Please add them to ApproximateBytes().

Reply:
This struct's life cycle limits in mounter and will be set to nil after here :https://github.com/pingcap/tiflow/blob/master/cdc/entry/mounter.go#L122, so it is no need to add them to `ApproximateBytes()`.